### DISCUSSION TITLE
100+ false positives on 2021-11-22 01:36 UTC

### DISCUSSION THREAD

**Discussion by andrewzwaniga** [chime-frb-open-data/community#41] [Announcements]
To all CHIME/FRB VOEvent Subscribers,

Please be advised that a system issue caused 100+ VOEvents to be published around **2021-11-22 01:36 UTC**. All VOEvents published with timestamps of **1970-01-06** are in fact false positives (non-FRBs) that erroneously triggered the VOEvent system. The issue has been resolved and you should now expect nominal behaviour from the Service. 

This type of system event is considered extremely rare; nonetheless, we are working to prevent recurrences of this type of event.

Thank you for your understanding and for continuing to use the CHIME/FRB VOEvent Service.

Best regards,
Andrew

**Comment by fabianschussler** [comment]
Are the hundreds of alerts that we got earlier today affected by a similar problem?
While the time is changing from one alert to the next, the date is clearly wrong.  eg. Timestamp [UTC]: 1970-01-03 09:23:22.268835+00:00

**Reply by andrewzwaniga** [reply]
Hi Fabian, thanks for writing. Yes it is a similar problem to last time. I have determined a hot fix that will prevent this from recurring and will be implementing it tomorrow. It is very likely that all of the alerts with a detection time from `1970` are false positives -- all of them should have been issued a retraction VOEvent as well by now, though the sheer volume of retractions can sometimes lead to delays in issuing those alerts. 

Best regards,
Andrew
